src/content/components/uikit/index.ts

```diff
@@ -67,10 +68,11 @@ const uikitComponents: Component[] = [
     spinConfig,
     switchConfig,
     {
-        id: 'table',
-        title: 'Table',
+        id: 'switch',
```

Comment:
Second switch with `isComingSoon: true` should be removed.
<img width="379" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/802c80af-4b9d-4fc3-9be8-8cb484102a2a">